scripts/data_guide/data_guide.qmd

```diff
@@ -0,0 +1,620 @@
+
+data_sv.txt
+
+* Description: Structural variant data
+* Details: [Structural Variant format](https://docs.cbioportal.org/file-formats/#structural-variant-data)
+
+genomic_information.txt
+
+* Description: File describing genomic coordinates covered by all platforms for GENIE data. Note: Not all centers contributed copy number data to GENIE
+* Details: This is not a cBioPortal file format.
+  * Chromosome, Start_Position, End_Position: Gene positions
+  * Hugo_Symbol: Re-mapped gene symbol based on gene positions
+  * ID: Center submitted gene symbols
+  * SEQ_ASSAY_ID: The institutional assay identifier for genomic testing platform. Feature_Type: "exon", "intron", or "intergenic"
+  * includeInPanel: Used to define gene panel files for cBioPortal. clinicalReported: These are the genes that were clinically Reported. Blank means information not provided.
+
+assay_information.txt
+
+* Description: Assay information
+* Details: This is not a cBioPortal file format
+  * is_paired_end, library_selection, library_strategy, platform, read_length, target_capture_kit, instrument_model: defined by [GDC read group](https://docs.gdc.cancer.gov/Data_Dictionary/viewer/#?view=table-definition-view&id=read_group)
+  * number_of_genes: Number of genes from which variants are called
+  * variant_classifications: List of types of variants that are reported for this assay.
+  * gene_padding: Number of base pairs to add to exon endpoints for the inBED filter.
+  * alteration_types: List of alteration types. specimen_type: List of specimen types.
+  * specimen_tumor_cellularity: Tumor Cellularity Cutoff
+  * calling_strategy: Tumor only or tumor normal.
+  * coverage: List of coverage
+
+data_cna_hg19.seg
+
+* Description: Segmented copy number data. Note: Not all centers contributed copy number data to GENIE
+* Details: [SEG format](https://docs.cbioportal.org/5.1-data-loading/data-loading/file-formats#segmented-data)
+
+
+<!-- : GENIE Data Files {tbl-colwidths="\[35,35,30\]"} This is the control table col widths-->
+
+# Clinical Data
+
+AGE_AT_SEQ_REPORT
+
+* Example Values: Integer values, <18, or >89
+* Data Description: The age of the patient at the time that the sequencing results were reported. Age is masked for patients aged 90 years and greater and for patients under 18 years.
+
+CENTER
+
+* Example Values: MSK
+* Data Description: The center submitting the clinical and genomic data
+
+ETHNICITY
+
+* Example Values: Non-Spanish/nonHispanic Spanish/Hispanic Unknown
+* Data Description: Indication of Spanish/Hispanic origin of the patient; this data element maps to the NAACCR v16, Element #190. Institutions not collecting Spanish/Hispanic origin have set this column to Unknown.
+
+ONCOTREE_CODE
+
+* Example Values: LUAD
+* Data Description: The primary cancer diagnosis "main type", based on the OncoTree ontology. The version of Oncotree ontology that was used for GENIE 13.0-public is oncotree_2021_11_02
```

Comment:
Update to use the release name parameter and to link out to the specific oncotree code